### ISSUE TITLE
Powershell files from official Microsoft Nuget Microsoft.PowerShell.SDK are not signed

### ISSUE THREAD

**Issue by prateekprshr-nith** [PowerShell/PowerShell#18247]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://github.com/PowerShell/PowerShell/releases)
- [X] Search the existing issues.
- [X] Refer to the [FAQ](https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md).
- [X] Refer to [Differences between Windows PowerShell 5.1 and PowerShell](https://docs.microsoft.com/powershell/scripting/whats-new/differences-from-windows-powershell).

### Steps to reproduce

Some psd1 files, which are a part of official Microsoft nuget package - [NuGet Gallery | Microsoft.PowerShell.SDK 7.2.6](https://www.nuget.org/packages/Microsoft.PowerShell.SDK/) are not signed. Specifically, all the files in `contentFiles/any/any/runtimes/unix` are missing signatures.

We are an internal MS team depending on this Nuget and are experiencing build failures due to missing signatures in the mentioned files.

### Expected behavior

```console
The files should be signed
```

### Actual behavior

```console
The files are not signed
```

### Environment data

```powershell
Name                           Value
----                           -----
PSVersion                      7.2.6
PSEdition                      Core
GitCommitId                    7.2.6
OS                             Microsoft Windows 10.0.22621
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by SteveL-MSFT** [comment]
@prateekprshr-nith those files are for non-Windows which doesn't have a way to validate signatures

**Comment by prateekprshr-nith** [comment]
@SteveL-MSFT, these files are pulled automatically for our .net 6 project since we depend on [NuGet Gallery | Microsoft.PowerShell.SDK 7.2.6](https://www.nuget.org/packages/Microsoft.PowerShell.SDK/). After our project is built, the scans in our pipeline report these files as malicious since they are not signed.
We have to account for this scenario as well. Maybe signatures in these files will not be validated when they are being used as it is in non-Windows, but that should be fine.

**Comment by prateekprshr-nith** [comment]
Hey @TravisEz13, Is there an ETA for the closure of this issue? We are completely blocked in our feature development because of this.

**Comment by prateekprshr-nith** [comment]
Hey @TravisEz13, @SteveL-MSFT, Is there an ETA for this issue? We are blocked because of this.

**Comment by adityapatwardhan** [comment]
@prateekprshr-nith The issue was fixed in version v7.3.0. The PR for it was here: https://github.com/PowerShell/PowerShell/pull/18449

v7.3.0 needs .NET 7. The fix for authenticode signed linux binaries will be coming to v7.2.x but not before January.

**Comment by pvsraviteja** [comment]
@adityapatwardhan any plans to get the binaries signed in January ?

**Comment by adityapatwardhan** [comment]
Yes it is planned for January.

**Comment by prateekprshr-nith** [comment]
@adityapatwardhan, is there an update on this?

**Comment by TravisEz13** [comment]
@prateekprshr-nith The PR to fix this was merged into 7.2.9, which has been released:
https://github.com/PowerShell/PowerShell/pull/18920#issuecomment-1402484464

**Comment by prateekprshr-nith** [comment]
Thanks for the update @TravisEz13 👍
